test/integration/evictions/evictions_test.go

```diff
@@ -351,21 +441,9 @@ func newPod(podName string) *v1.Pod {
 	}
 }
 
-func addPodConditionSucceeded(pod *v1.Pod) {
-	pod.Status = v1.PodStatus{
-		Phase: v1.PodSucceeded,
-		Conditions: []v1.PodCondition{
-			{
-				Type:   v1.PodReady,
-				Status: v1.ConditionTrue,
-			},
-		},
-	}
-}
-
-func addPodConditionReady(pod *v1.Pod) {
+func addPodConditionReady(pod *v1.Pod, phase v1.PodPhase) {
```

Comment:
The phase has nothing to do with the condition Ready, so I would put set it outside of the function, where needed.

```suggestion
func addPodConditionReady(pod *v1.Pod) {
```

Reply:
done